model/database/deprecated/metrics-deprecated.yaml

```
@@ -77,7 +89,9 @@ groups:
     type: metric
     metric_name: db.client.connections.create_time
     stability: development
-    deprecated: "Replaced by `db.client.connection.create_time`. Note: the unit also changed from `ms` to `s`."
+    deprecated:
+      reason: uncategorized
+      note: Replaced by `db.client.connection.create_time` with a `s` unit.
```

(and same below if you take this suggestion)
```suggestion
      note: Replaced by `db.client.connection.create_time` with unit `s`.
```